package.json

```diff
@@ -20,28 +29,30 @@
   },
   "homepage": "https://github.com/nqminds/nqm-api-tdx",
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-eslint": "^7.2.3",
-    "babel-loader": "^7.0.0",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-preset-env": "^1.7.0",
-    "chai": "^3.5.0",
-    "docdash": "^1.1.1",
-    "eslint": "^3.19.0",
-    "eslint-loader": "^1.7.1",
-    "eslint-plugin-babel": "^4.1.1",
-    "eslint-plugin-jest": "^22.14.1",
-    "eslint-plugin-mocha": "^4.9.0",
-    "jest": "^23.6.0",
-    "jsdoc": "^3.6.3",
-    "lodash": "^4.17.15",
-    "webpack": "^2.5.1"
+    "@babel/cli": "^7.13.14",
+    "@babel/eslint-parser": "^7.13.8",
+    "@babel/preset-env": "^7.13.12",
+    "babel-loader": "^8.2.2",
+    "babel-plugin-add-module-exports": "^1.0.4",
+    "chai": "^4.3.4",
+    "docdash": "^1.2.0",
+    "eslint": "^7.23.0",
+    "eslint-loader": "^4.0.2",
+    "eslint-plugin-babel": "^5.3.1",
+    "eslint-plugin-jest": "^24.3.2",
+    "eslint-plugin-mocha": "^8.1.0",
+    "jest": "^27.0.0-next.6",
+    "jsdoc": "^3.6.6",
+    "lodash": "^4.17.21",
+    "npm-run-all": "^4.1.5",
```

Comment:
A ~100kb library to do this? :disappointed_relieved: 

Next thing you'll be using lodash map...

Reply:
And it has 9 dependencies as well! I mainly didn't want to write `npm run build && npm run build:dev` since I was worried about the `&&` on non-Linux platforms, and hard-coding `npm` in case someone wants to use `yarn`.

However, it looks like https://www.npmjs.com/package/yarn-or-npm is tiny, and has much less dependencies, and `&&` is supported by both Unix and Windows, and I doubt there are any other OSes anybody is using Node with, so I could switch to using https://www.npmjs.com/package/yarn-or-npm instead.

Reply:
Supporting windows is undesirable :)

Reply:
Awesome, I've switched to `yarn-or-npm`, so we still have both Windows and Yarn supported (and removed like 2000 lines from our `package-lock.json` file).

Supporting Windows isn't too much trouble though, just for the off chance that one of the Windows users (@nallott or Uni of Soton) decides to install via GitHub, not npm. At least we don't need to worry about esoteric platforms like the [Unisys 1100/2200](https://begriffs.com/posts/2018-11-15-c-portability.html) where one `char` is 9 bits, and has a +0 and -0.